eclass/elisp.eclass

```diff
@@ -166,7 +166,11 @@ elisp_src_test() {
 elisp_src_install() {
 	elisp-install ${PN} *.el *.elc
 	if [[ -n ${SITEFILE} ]]; then
-		elisp-site-file-install "${FILESDIR}/${SITEFILE}"
+		if [[ -f "${FILESDIR}/${SITEFILE}" ]]; then
+			elisp-site-file-install "${FILESDIR}/${SITEFILE}"
+		else
+			elisp-make-site-file "${SITEFILE}"
+		fi
```

Comment:
so it install the siftefile if it exists and it makes it if it does not? Where ins installation in 2nd case?

Reply:
```bash
elisp-make-site-file "${SITEFILE}"  # => ${T}/${SITEFILE}
elisp-site-file-install "${T}/${SITEFILE}"
```

Reply:
> I just dont understand why make-sitefile also installs it.

What else would one do with it, if not install it? It's similar to `make_desktop_entry()` in desktop.eclass.

Reply:
We could of course go for a different name if `elisp-make-site-file` is misleading (then again, the name is similar to `make_desktop_entry`).

Reply:
I think make_desktop_entry makes more sense because we also have domenu. Because this eclass doe not have it I think it is fine.